frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/utils.ts

```diff
@@ -265,16 +265,25 @@ const removeListRunAfters = (task: PipelineTask, listIds: string[]): PipelineTas
   return task;
 };
 
-const removeEmptyDefaultParams = (task: PipelineTask): PipelineTask => {
+export const removeEmptyFormFields = (task: PipelineTask): PipelineTask => {
+  let output = task;
```

Comment:
nit: You could use a more descriptive naming as we are already dealing with `resource.output` in the same function.
One more question, instead of 
`let output = task ( object pointing)` should we make it
`let trimmedTask = { ...task, params }` in line 273 directly?

Reply:
Thanks @abhinandan13jan I changed `output` to `trimmedTask`.

In line 273 I could not change to `let trimmedTask` because I need access to this variable also in the other if statements.